example/bzlmod/mylib/mylib_test.cc

```diff
@@ -0,0 +1,14 @@
+#include <gtest/gtest.h>
+#include <string>
+#include "mylib.h"
```

Comment:
So this is a personal preference thing, and maybe things have changed since I last used C++ to make this irrelevant, but I got in the habit of putting my local `#include` files first. The idea was to make sure they already `#include`d everything they need, so you don't lock their dependents into `#include`ing things they might not otherwise directly depend upon to keep the build working. But you can take that preference or leave it.